Added forms and footer section

sireclifford opened the pull request:
- Added forms section specifically using the necessary html elements.
- Changed necessary links to buttons.
- Added the different button styles with their actions

chelmerrox committed:
Add Contact form & Footer sections

Commit:
added contact section and footer

Commit:
Implemented form tag and related elements in the form section

Commit:
refactored the css to include different types of Button

Commit:
refactored css to take off duplicate selectors

Commit:
refactored css to fix linters

ishimwezachee commented on index.html:
```diff
@@ -165,8 +167,72 @@ <h1>Skills</h1>
+                <h1>Getting Started</h1>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                    labore
+                    et dolore magna.</p>
+                <form action="https://formspree.io/f/123" method="post">
+                    <div class="text-input user-name">
+                        <label for="user-name">Your name</label>
+                        <input type="text" name="name" placeholder="John Doe" id="user-name" required maxlength="30" />
+                    </div>
+                    <div class="text-input user-email">
+                        <label for="user-email">Your email</label>
+                        <input type="email" name="email" placeholder="john.doe@mail.com" id="user-email" />
+                    </div>
+                    <div class="text-input user-message">
+                        <label>
+                            <textarea id="msg" name="user-message" placeholder="Enter your message here..."
+                                maxlength="500"></textarea>
+                        </label>
+                    </div>
+                    <div class="collab">
+                        <button type="button" class="btn-primary-dark">Start Collaboration</button>
+                    </div>
```
- [x] You should give your button a type of submit so that you can be able to submit you form. Please change you `type="button" to type="submit"`

ishimwezachee replied:
- [x] I have changed your button type to submit to see if your formservices was configured correctly, but I received the error. Please fix the issue so that when I send data with you form, I will receive a successful message. Here is how it look like when I try to submit some data in your form.
![image](https://user-images.githubusercontent.com/43877689/171446601-b230d6fb-3280-4a11-8320-619cd5123cb5.png)

ishimwezachee commented on index.html:
```diff
@@ -165,8 +167,72 @@ <h1>Skills</h1>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                    labore
+                    et dolore magna.</p>
+                <form action="https://formspree.io/f/123" method="post">
+                    <div class="text-input user-name">
+                        <label for="user-name">Your name</label>
+                        <input type="text" name="name" placeholder="John Doe" id="user-name" required maxlength="30" />
+                    </div>
+                    <div class="text-input user-email">
+                        <label for="user-email">Your email</label>
+                        <input type="email" name="email" placeholder="john.doe@mail.com" id="user-email" />
+                    </div>
+                    <div class="text-input user-message">
+                        <label>
+                            <textarea id="msg" name="user-message" placeholder="Enter your message here..."
+                                maxlength="500"></textarea>
+                        </label>
+                    </div>
+                    <div class="collab">
+                        <button type="button" class="btn-primary-dark">Start Collaboration</button>
```
- [x] All Your input fields should have the `required `attribute to activate the validation on submit. please add a `required `attribute so that you form `inputs `will be validated, or user can not send an empty fields .

ishimwezachee reviewed (changes requested):
Hi @sireclifford and @chelmerrox  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)

### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Ishimwe Zachee  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
(Not automatically expanded because 3 MB is too large. You can [expand it anyway](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966) or [open it in a new window](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966).)

Commit:
Implemented changes suggested by code reviewer
Added the correct form link to recieve the form data
Changed form button type from button to submit

danmainah reviewed (approved):
Hi @sireclifford and @chelmerrox , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
